components/ProfilePageComponent/SkillsSection.tsx

```diff
@@ -39,6 +47,14 @@ const SkillsSection = () => {
         setIsOpen(false);
     }
 
+    useEffect(() => {
+        GetData('/api/skills?userId=' + _id).then((res: any) => {
+            console.log(res);
```

Comment:
@JRakin bhai useCallback along with useEffect. you can look into experience section as an example